### PR TITLE
i18n(fr): Add `deploy/cloudflare.mdx` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)

i18n(fr): Add `deploy/cloudflare.mdx` in French language

I am participating in the Hacktoberfest! 🎃
- [i18n(fr): Add deploy/cloudflare.mdx in French language](https://github.com/withastro/docs/commit/fb3897ac1d575b57bf00aeffc5a784544f42c40c)

Related issues & labels (optional)

Related to this issue https://github.com/withastro/docs/issues/4984, don't close it.

### CODE REVIEW

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

File: src/content/docs/fr/guides/deploy/cloudflare.mdx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+ - **Framework preset**: `Astro`
+ - **Build command:** `npm run build`
+ - **Build output directory:** `dist`
+
+7. Cliquez sur le bouton **Enregistrer et déployer**.
+
+## Comment déployer un site à l'aide de Wrangler
+
+1. Installez [Wrangler CLI] (https://developers.cloudflare.com/workers/wrangler/get-started/).
```
Comment (Yan-Thomas) [inline src/content/docs/fr/guides/deploy/cloudflare.mdx:40]:
```suggestion
1. Installez [Wrangler CLI](https://developers.cloudflare.com/workers/wrangler/get-started/).
```

File: src/content/docs/fr/guides/deploy/cloudflare.mdx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+Une fois vos ressources téléchargées, Wrangler vous donnera une URL de prévisualisation pour inspecter votre site. Lorsque vous vous connectez au tableau de bord de Cloudflare Pages, vous verrez votre nouveau projet.
+
+### Activer la prévisualisation localement avec Wrangler
+
+Pour que la prévisualisation fonctionne, vous devez installer `wrangler`
+
+``bash
+pnpm install wrangler --save-dev
+```
```
Comment (Yan-Thomas) [inline src/content/docs/fr/guides/deploy/cloudflare.mdx:64]:
````suggestion
```bash
pnpm install wrangler --save-dev
```
````

File: src/content/docs/fr/guides/deploy/cloudflare.mdx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+Suivez les étapes ci-dessous pour configurer l'adaptateur. Vous pouvez ensuite déployer en utilisant l'une ou l'autre des approches documentées ci-dessus.
+
+### Installation rapide
+
+Ajoutez l'adaptateur Cloudflare pour activer SSR dans votre projet Astro avec la commande `astro add` suivante. Cela installera l'adaptateur et apportera les changements appropriés à votre fichier `astro.config.mjs` en une seule étape.
+
+``bash
+npx astro add cloudflare
+```
```
Comment (Yan-Thomas) [inline src/content/docs/fr/guides/deploy/cloudflare.mdx:85]:
````suggestion
```bash
npx astro add cloudflare
```
````

File: src/content/docs/fr/guides/deploy/cloudflare.mdx [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+### Installation manuelle
+
+Si vous préférez installer l'adaptateur manuellement, suivez les deux étapes suivantes :
+
+1. Ajoutez l'adaptateur `@astrojs/cloudflare` aux dépendances de votre projet en utilisant votre gestionnaire de paquets préféré. Si vous utilisez npm ou si vous n'êtes pas sûr, lancez ceci dans le terminal :
+
+ ``bash
+ npm install @astrojs/cloudflare
+    ```
```
Comment (Yan-Thomas) [inline src/content/docs/fr/guides/deploy/cloudflare.mdx:95]:
````suggestion
 ```bash
 npm install @astrojs/cloudflare
 ```
````

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps. There was a few broken code blocks/link but I fixed them already 🙌

**Comment by thomasbnt** [comment]
Owwh thanks @Yan-Thomas for fixed these lines!

**Pull request merged**